spec/initializers/settings_schema_spec.rb

```diff
@@ -24,7 +24,7 @@
 
     context 'jwt' do
       it 'expiration_hours is nil' do
-        settings[:jwt][:expiration_hours] = nil
+        settings[:jwt] = settings[:jwt].merge(expiration_hours: nil)
```

Comment:
The same needs to be done [here](https://github.com/ontohub/ontohub-backend/blob/b458c9e30a6606281a08edcb96a50cacfa78fe8e/spec/initializers/settings_schema_spec.rb#L147).